package.json

```diff
@@ -126,7 +126,7 @@
     "vite-plugin-html": "^3.2.0",
     "vite-plugin-node-polyfills": "^0.7.0",
     "vite-plugin-sentry": "^1.1.6",
-    "vitest": "^0.28.1",
+    "vitest": "^0.28.3",
```

Comment:
I think normally it's better to update packages by separate pr, but seems it's ok this time